src/planwise/component/projects2.clj

```diff
@@ -49,8 +49,8 @@
 
 (defn start-project
   [store project-id]
-  (db-start-project! (get-db store) {:id project-id})
   (let [project (get-project store project-id)]
+    (db-start-project! (get-db store) {:id project-id})
```

Comment:
Any reason for this change? This may in fact make the later call to `create-initial-scenario` with an outdated version of the project.